core/src/main/java/com/amazon/opendistroforelasticsearch/sql/planner/logical/LogicalPlanDSL.java

```diff
@@ -83,6 +84,16 @@ public static LogicalPlan dedupe(
         input, Arrays.asList(fields), allowedDuplication, keepEmpty, consecutive);
   }
 
+  public static LogicalPlan rareTopN(LogicalPlan input, Boolean rareTopFlag,
+      List<Expression> groupByList, Expression... fields) {
+    return rareTopN(input, rareTopFlag, 10, groupByList, fields);
```

Comment:
Some use a constant like DEFAULT_NO_OF_RESULTS and some have 10 hardcoded. Should probably be consistent with the constant.

Reply:
Used the variable/function to get default value wherever required. hardcoded values are used for the test purpose.